index.bs

```diff
@@ -24,6 +24,68 @@ Assume Explicit For: yes
 Blah blah blah.
 </section>
 
+<h2 id="deployed-mitigations">
+Some browsers have deployed and announced protections against
+navigational-tracking.  This section is a work in progress to detail what
+protections have been shipped and / or are planned. This section should not
+be understood to be comprehensive.
+
+<h3>Safari</h3>
+Safari uses a heuristic-based approach to combat navigational-tracking. Safari uses
+a client-side machine classifier to determine which sites (determined by
+eTLD+1) are participating in navigational-tracking. The main inputs to this
+classifier are "<a href="//webkit.org/blog/7675/intelligent-tracking-prevention/">
+subresource under number of unique domains, sub frame under number of unique
+domains, and number of unique domains redirected to</a>".
+
+Once a site is classified as a site with cross-site tracking capabilities, it
+can "taint" other sites it participates in redirection chains with.  For
+example, consider the case of a user clicks on a link on
+<code>start.example</code>, which redirects to <code>second.example</code>,
+which redirects to <code>third.example</code>, which redirects to
+<code>fourth.example</code>, which redirects to <code>end.example</code>. If
+Safari has classified <code>third.example</code> as having tracking
+capabilities, the above behavior may result in Safari also classifying
+<code>second.example</code> and <code>fourth.example</code> has having tracking
+capabilities.
```

Comment:
A classified domain can not taint another by redirecting to it. Instead an unclassified domain can be tainted by redirecting to a classified domain. Transitive classification happens to the domain that makes the redirect decision.